incubator-tools/bank_statement_post_processing_tool/bank_statement_post_processing_tool.ipynb

```diff
@@ -0,0 +1,2834 @@
+{
```

Comment:
<p>There are links/customer names that shouldn't be included in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
redacted those details in updated notebook